app/javascript/stylesheets/general.scss

```diff
@@ -0,0 +1,711 @@
+//HOMEPAGE
+.hero-img {
+  background-image: linear-gradient(
+      rgba(37, 38, 54, 0.2),
+      rgba(37, 38, 54, 0.2)
+    ),
+    url("./../images/banner.svg");
+  height: 45rem;
+  background-position: left;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  @include mappy-bp(small large) {
+    height: 34.375rem;
+  }
+
+  .hero-title {
+    color: $white-color;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 4rem;
+    line-height: 5rem;
+    display: table-caption;
+    margin-left: 1rem;
+    margin-top: 19rem;
+  }
+}
+
+.home-title {
+  color: $dark-color;
+  font-weight: 600;
+  font-size: 2rem;
+
+  @include mappy-bp(small large) {
+    font-size: 1.313rem;
+    margin-left: 1rem;
+  }
+}
+
+.new-classes {
+  margin-left: 9rem;
+  padding-top: 7rem;
+
+  @include mappy-bp(small large) {
+    margin-left: 1rem;
+    padding-top: 5rem;
+  }
+}
+.filter-wrapper {
+  width: 100%;
+  display: flex;
+  overflow-x: auto;
+  margin: 0.75rem 0;
+
+  &::-webkit-scrollbar {
+    width: 0;
+  }
+}
+
+.new-classes-wrapper {
+  width: 100%;
+  display: flex;
+  overflow-x: auto;
+
+  &::-webkit-scrollbar {
+    width: 0;
+  }
+}
+
+.categories {
+  margin-left: 9rem;
+  padding-top: 7rem;
+
+  @include mappy-bp(small large) {
+    margin-left: 1rem;
+    padding-top: 5rem;
+  }
+
+  .categories-wrapper {
+    display: grid;
+    grid-gap: 1rem;
+
+    @include mappy-bp(small large) {
+      grid-template-columns: repeat(2, 10.25rem);
+      margin-top: 1rem;
+    }
+  }
+}
+
+.popular {
+  margin-left: 9rem;
+  padding-top: 7rem;
+
+  @include mappy-bp(small large) {
+    margin-left: 0;
+    padding-top: 5rem;
+  }
+
+  .popular-wrapper {
+    @include mappy-bp(small large) {
+      margin-left: 1rem;
+    }
+  }
+}
+
+//////////////////////////////////////////////////////////////////////////
+//NAVBAR
+.navbar {
+  background: $primary-color;
+  height: 5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.2rem;
+  position: sticky;
+  top: 0;
+  z-index: 999;
+
+  .navbar-container {
+    display: flex;
+    justify-content: space-between;
+    height: 5rem;
+    z-index: 1;
+    width: 100%;
+    max-width: 81.25rem;
+    margin-right: auto;
+    margin-left: auto;
+    padding-left: 3.1rem;
+
+    .navbar-logo {
+      color: $white-color;
+      justify-self: start;
+      cursor: pointer;
+      margin-top: 2%;
+
+      @include mappy-bp(small large) {
+        position: absolute;
+        top: 0;
+        left: 0;
+        transform: translate(25%, 80%);
+        margin-top: 0;
+      }
+    }
+
+    .navbar-menu-icons {
+      color: $white-color;
+
+      @include mappy-bp(small large) {
+        display: flex;
+        position: absolute;
+        top: 50%;
+        left: 90%;
+        transform: translate(-100%, -50%);
+        font-size: 1.8rem;
+      }
+    }
+    .navbar-menu-icon-notification {
```

Comment:
**issue (blocking)**: all this indentation is unecessary and just creates overly specific css classes. Each new class should be in the global scope (not indented), the only things you should indent are `:hover`, `.is-active`, `mappy-bp`